package.json

```diff
@@ -100,5 +106,28 @@
   "files": [
     "dist/**/!(*.test).js",
     "dist/node/index.d.ts"
-  ]
+  ],
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
```

Comment:
Here I think we can modify this to be closer to the browsers that are supported by the SDK. If you run `npx browserslist` in the terminal, it'll show you which browsers are supported by this configuration. The config that's here supports `ie 11`, which seems to increase the bundle size a bit, probably by including polyfills that modern browsers don't need. 

Something like this seems to be close to what we support:
```
"> 1%, not IE 11",
"last 2 and_ff version",
"not op_mini all, not opera > 0, not samsung > 0"
```

`npx browserslist` will show you what is included with this syntax. There's more information on the syntax in the [docs](https://github.com/browserslist/browserslist).

Reply:
that's a good point about ie 11!